interfaces/builtin/system_observe.go

```diff
@@ -117,6 +117,16 @@ dbus (send)
     interface=org.freedesktop.DBus.Peer
     member=GetMachineId
     peer=(label=unconfined),
+
+# Allow reading if protected hardlinks are enabled, but don't allow enabling or
+# disabling them
+@{PROC}/sys/fs/protected_hardlinks r,
+@{PROC}/sys/fs/protected_symlinks r,
+@{PROC}/sys/fs/protected_fifos r,
+@{PROC}/sys/fs/protected_regular r,
+
+# Allow reading what size hugepages have been configured on a system.
+/sys/kernel/mm/hugepages/ r,
```

Comment:
This is already allowed in hardware-observe.

Reply:
ack removed, greengrass can just plug hardware-observe instead